TestSwitchApi/TestSwitchApi/Controllers/SessionsController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Net;
+using Microsoft.AspNetCore.Cors;
+using Microsoft.AspNetCore.Mvc;
+using TestSwitchApi.Models.DataModels;
+using TestSwitchApi.Models.Request;
+using TestSwitchApi.Repositories;
+
+namespace TestSwitchApi.Controllers
+{
+    [ApiController]
+    [Route("/sessions")]
+    [EnableCors("AllowOrigin")]
```

Comment:
shouldn't need this `EnableCors` attribute

Reply:
removed EnableCors